File: 1528-shuffle-string/1528-shuffle-string.js
/**
 * @param {string} s
 * @param {number[]} indices
 * @return {string}
 */
var restoreString = function(s, indices) {
    let t = []
    for(let i = 0 ; i < s.length ; i++){
    t[indices[i]]=   s[i]
        }
    return t.join('')
};

File: 1528-shuffle-string/1528-shuffle-string.java
class Solution {
    public String restoreString(String s, int[] indices) {
        char[] newS = new char[s.length()];
            
        for(int i = 0 ; i < s.length(); i++){
            newS[indices[i]] = s.charAt(i);
        }
        
        return String.valueOf(newS);
        
    }
}